### ISSUE TITLE
Is this applicable for a device running LEDE with a single ethernet port? 

### ISSUE THREAD

**Issue by wicadmin** [initialed85/openwrt_or_lede_config#1]
My WAN is an LTE card, my VLANs need to trunk to a managed switch via a single ethernet port. Do I configure it the same way with the 'switch' info in the config file since the device has no physical switch ports?

Also, no wifi is needed on this device (running LEDE).

**Comment by initialed85** [comment]
Hi mate,

I believe the "switch" piece in the network config is only applicable to devices with a switch chipset onboard- it sounds like your device doesn't (but I think you can confirm with "swconfig list" or something like that).

However I do think some of the config from "dot1q_fed_to_vaps" will work for you; let's say for your use case you want the following:

- LTE WWAN to be VLAN2
- Device management to be VLAN3
- Trunk to the managed switch

I think you can do something like this in /etc/config/network (assumption- wwan0 can be added to a bridge):

	# we'll bridge wwan0 and eth1.2 together for vlan2
	config interface 'vlan2'
	    option ifname 'eth1.2 wwan0'
	    option type 'bridge'
	    option proto 'dhcp'  # see note below

	# we'll use vlan3 as the management
	config interface 'vlan3'
	    option ifname 'eth1.7'
	    option proto 'static'
	    option ipaddr '192.168.1.2'
	    option netmask '255.255.255.0'

Regarding the use of DHCP on the VLAN2 bridge- if you use something like ModemManager to normally set the IP of the wwan0 interface (or if you're using PPP and end up with a ppp0 interface) then you might need to do some reading on how to have that process remove the IP from the wwan0/ppp0 interface and set it on the VLAN2 interface instead (that's typical behaviour with bridging- once you set up the bridge, the bridge itself holds the IP for all the bridged interfaces)

Best of luck buddy!

**Comment by wicadmin** [comment]
Hey...first thanks for the very detailed reply. 
On my x86_64 LEDE target, it doesn't seem 'swconfig list' is there. But it is rather certain this device (Chromebox) doesn't have an on-board switch chipset.

What I think is missing and I was hoping to get from your reply most of all was how to configure the trunk.

**Comment by initialed85** [comment]
Ah, I see- well in your instance I'm not sure, as I'm pretty sure that's handled with the switch config statements.

Maybe if you install all the switch stuff you'll be able to point it at your normal ethernet port and treat it like a switch with only one port?

Alternately, it might just work as is- I *think* on Ubuntu just having a sub-interface (e.g. eth0.2) results in the tagging of that traffic as VLAN2, so maybe OpenWrt/LEDE will be the same?

**Comment by wicadmin** [comment]
Yes, indeed...that was implied on some reading I did but I was not sure it would fly. I'll give it a go and check it out.

Thanks for the items in this repo.
